Assets/HoloToolkit/Input/Scripts/GazeManager.cs

```diff
@@ -129,5 +129,17 @@ private void UpdateStabilizationPlane()
                 HolographicSettings.SetFocusPointForFrame(newFocusPointPosition, -gazeDirection);
             }
         }
+
+        /// <summary>
+        /// Sends message to the object currently focused on.
+        /// </summary>
+        /// <param name="message">Message to send</param>
+        public void SendMessageToFocusedObject(string message)
```

Comment:
@NeerajW Good point, I mentioned this in the forum thread. Exposing FocusedObject would create an additional dependency potentially used everywhere outside of GazeManager. This is probably fine, but PR intent was to minimize side effects and prevent the abstraction leaking outside of GazeManager.